test/components/contextMenuTest.js

```diff
@@ -122,7 +122,7 @@ describe('ContextMenu', function () {
         .keys('\uE007')
         .pause(10)
         .tabByUrl(this.formfill)
-        .getValue(this.input).should.eventually.be.equal(this.values[3])
+        .getValue(this.input).should.eventually.be.equal(this.values[2])
     })
 
     it('check left/right on non sub menu item', function *() {
```

Comment:
The puase() calls should not be needed by the way, you should instead wait on some element state like a class name. The pause will fail intermittently if it is required.   It was already there though so I'll merge this.  Thanks!